sunpy/map/sources/stereo.py

```diff
@@ -55,6 +56,24 @@ def rsun_arcseconds(self):
         """
         return self.meta.get('rsun', None)
 
+    @property
+    def rsun_obs(self):
+        """
+        Radius of the sun in arcseconds as a quantity.
+
+        References
+        ----------
+        https://sohowww.nascom.nasa.gov/solarsoft/stereo/secchi/doc/FITS_keywords.pdf
+        """
+        rsun_arcseconds = self.meta.get('rsun', None)
+
+        if rsun_arcseconds is None:
+            warnings.warn("Missing metadata for solar radius: assuming photospheric limb as seen from Earth.",
```

Comment:
Rather than doing this here why not just call out to `GenericMap`

```suggestion
            rsun_arcseconds = super().rsun_obs
```

Reply:
Done.